corpus/transform.py

```diff
@@ -491,3 +491,73 @@ def run(self):
 
         with open(str(self.out_segments), "w") as segments_outfile:
             segments_outfile.writelines(segment_file_names)
+
+
+class LexiconStrategy(enum.Enum):
+    PICK_FIRST = 0
+
+
+class ApplyLexiconToTranscriptionsJob(Job):
+    """
+    Use a bliss lexicon to convert all words in a bliss corpus into their phoneme representation.
+
+    Currently only supports picking the first phoneme.
+    """
+
+    def __init__(
+        self,
+        bliss_corpus,
+        bliss_lexicon,
+        word_separation_orth,
```

Comment:
```suggestion
        word_separation_orth=None,
```
maybe make None the default.

Reply:
But "None" would not give valid results... one could use `" "` but I did not want to add that so that people have to think about what they need.

Reply:
Ok, will add `None`